src/app/(proper_react)/(onboarding)/user/welcome/GetStarted.tsx

```diff
@@ -7,13 +7,14 @@
 import Image from "next/image";
 import { useOverlayTriggerState } from "react-stately";
 import { useOverlayTrigger } from "react-aria";
-import styles from "./GetStarted.module.scss";
 import howItWorksHero from "./images/welcome-how-it-works.svg";
 import { useL10n } from "../../../../hooks/l10n";
 import { ModalOverlay } from "../../../../components/client/dialog/ModalOverlay";
 import { Dialog } from "../../../../components/client/dialog/Dialog";
 import { Button } from "../../../../components/server/Button";
 
+import viewStyles from "./View.module.scss";
```

Comment:
I get nervous about this, because it negates the whole benefit of using CSS modules: being able to modify or delete styles and knowing for sure that it won't affects code outside a particular component. Without it, CSS files tend to turn append-only, as folks get scared what can happen if they change existing CSS (at least, I do).

Two potential solutions:

1. Just copy-paste the styles.
2. Extract the common elements into a separate component and incorporate that into both steps.

Personally, I tend to [lean Write Everything Twice](https://kentcdodds.com/blog/aha-programming), i.e. copy-pasting - I don't think there would be large disadvantages to doing that here. But I could live with a separate component.

Reply:
Done in [fe0f99e](https://github.com/mozilla/blurts-server/pull/3208/commits/fe0f99e62d8353e5e8540e0599bbdd91a553c069).